src/typeDefs/mutations.ts

```diff
@@ -79,6 +79,8 @@ export const mutations = gql`
 
     forgotPassword(data: ForgotPasswordData!): Boolean!
 
+    otpCheck(data: ForgotPasswordData!): Boolean!
+
```

Comment:
I guess you should create a new input typedef named ```EmailVerificationData``` and add it here, instead of this. This would help in debugging in future.